common/src/test/java/com/graphicsfuzz/common/util/FragmentShaderToShaderJobTest.java

```diff
@@ -45,16 +44,17 @@ public void basicJsonGenerationTest() throws Exception {
     + "\n"
     + "in float in_f;\n"
     + "in vec2 in_v2;\n"
-    + "in vec3 in_v3;\n"
+    + "layout(location = 78) in vec3 in_v3;\n"
```

Comment:
Add a comment that this '78' is expected to be changed to a smaller value.

Reply:
done